packages/composables/src/useOrderDetails.ts

```diff
@@ -192,7 +193,9 @@ export function useOrderDetails(
   const shippingCosts = computed(() => _sharedOrder.value?.shippingTotal);
   const subtotal = computed(() => _sharedOrder.value?.price?.positionPrice);
   const total = computed(() => _sharedOrder.value?.price?.totalPrice);
-  const status = computed(() => _sharedOrder.value?.stateMachineState?.name);
+  const status = computed(
```

Comment:
Translations are not part of the business logic. We should use "core" values in the composables. A better solution might be to return the whole status object, where the translated value can be read on the template. What do you think?

Reply:
cool, let's solve that here, because I don't see any problem of using built-in property in the composable as well as our composable function do rely on the context, including a language.

I'm not entirely convinced yet, because in this particular case, we have translated property value that comes directly from API according to session context language - which is related to - so it is a part of the core logic.
in the backend it's called `TranslatedField` class and the values come for the specific snippet built-it. 

For custom/manual translation in the frontend project I would use - the `statusTechnicalName` + vue-demo-store snippet translation because it's probably out of language context and can be used as a translation key.